modules/mobile/spec/request/triage_teams_request_spec.rb

```diff
@@ -2,20 +2,34 @@
 
 require 'rails_helper'
 require_relative '../support/helpers/sis_session_helper'
-require_relative '../support/helpers/mobile_sm_client_helper'
 
 RSpec.describe 'Mobile Triage Teams Integration', type: :request do
-  include Mobile::MessagingClientHelper
   include SchemaMatchers
 
-  let!(:user) { sis_user(:mhv, mhv_correlation_id: '123', mhv_account_type:) }
+  let!(:user) { sis_user(:mhv, mhv_correlation_id: '123', mhv_account_type: 'Premium') }
 
-  before do
-    allow(Mobile::V0::Messaging::Client).to receive(:new).and_return(authenticated_client)
+  before { Timecop.freeze(Time.zone.parse('2017-05-01T19:25:00Z')) }
+
+  after { Timecop.return }
+
+  context 'when not authorized' do
+    it 'responds with 403 error' do
+      VCR.use_cassette('mobile/messages/session_error') do
+        get '/mobile/v0/messaging/health/recipients', headers: sis_headers
+      end
+      expect(response).not_to be_successful
+      expect(response).to have_http_status(:forbidden)
+    end
   end
 
-  context 'Premium User' do
-    let(:mhv_account_type) { 'Premium' }
+  context 'when authorized' do
+    before do
+      VCR.insert_cassette('sm_client/session')
```

Comment:
VCR by default will only allow a cassette to be played once so we know there isn't a redundant call to sessions since the authorization call is cached and used for the main service call.